implementation_design.md

```diff
@@ -1,214 +1,69 @@
-
-    /// @notice Only the auctioneer can call this method, passing in the two highest bids.
-    ///         The auction contract will verify the signatures on these bids,
-    ///         and that both are backed by funds deposited in the auction contract.
-    ///         Then the auction contract will deduct the second-highest bid amount
-    ///         from the account of the highest bidder, and transfer those funds to
-    ///         an account designated by governance, or burn them if governance
-    ///         specifies that the proceeds are to be burned.
-    ///         auctions are resolved by the auctioneer before the end of a current round
-    ///         at some time T = AUCTION_CLOSING_SECONDS where T < ROUND_DURATION_SECONDS.
-    /// @param bid1 the first bid
-    /// @param bid2 the second bid
-    function resolveAuction(Bid calldata bid1, Bid calldata bid2) external;
-}
-```
+The latest Solidity interface for the express lane auction contract can be found [here](https://github.com/OffchainLabs/nitro-contracts/pull/214/files). The latest Go implementation is [here](https://github.com/OffchainLabs/nitro/compare/express-lane-timeboost?expand=1)
 
 # Offchain Components
 
 ## Auctioneer API
 
-The auctioneer must expose a new RPC namespace called `timeboost` and a RPC method called `timeboost_submitBid`
+The auctioneer must expose a new RPC namespace called `auctioneer` and a RPC method called `timeboost_submitBid`
 
-### `timeboost_submitBid`
+### `auctioneer_submitBid`
 
 ```json
 {
   "type": "object",
   "properties": {
-    "chain_id": {
-      "type": "uint64",
+    "chainId": {
+      "type": "bigInt",
       "description": "chain id of the target chain"
     },
-    "address": {
-      "type": "string",
-      "description": "0x-prefixed, 20-byte address of the bidder"
+    "expressLaneController": {
+      "type": "address",
+      "description": "hex string of the desired express lane controller address"
+    },
+    "auctionContractAddress": {
+      "type": "address",
+      "description": "hex string of the auction contract address that the bid corresponds to"
     },
     "round": {
       "type": "uint64",
-      "description": "round number (0-indexed) for the round being bid on"
+      "description": "round number (0-indexed) for the round the bidder wants to become the controller of"
     },
     "amount": {
-      "type": "uint256",
-      "description": "The amount in wei of the deposit ERC-20 to bid"
+      "type": "bigInt",
+      "description": "The amount in wei of the deposit ERC-20 token to bid"
     },
     "signature": {
-      "type": "string",
-      "description": "0x-prefixed ECDSA signature of an ABI encoded tuple (domainValue, chainId, round, amount)"
+      "type": "bytes",
+      "description": "Ethereum signature over the bytes encoding of (keccak256(TIMEBOOST_BID), padTo32Bytes(chainId), auctionContractAddress, uint64ToBytes(round), padTo32Bytes(amount), expressLaneController)"
```

Comment:
Can we say which EIP defined the signature format, so there is no ambiguity here?